src/backend/expungeservice/expunger/analyzers/time_analyzer.py

```diff
@@ -71,6 +72,22 @@ def _mark_as_time_ineligible(charges, reason, eligibility_date):
         for charge in charges:
             charge.set_time_ineligible(reason, eligibility_date)
 
+    @staticmethod
+    def _set_arrests_as_eligible(expunger):
+        # If most recent conviction is time eligible
+        # AND all other charges are arrests
+        # Then make all other charges time eligible
+        if expunger.most_recent_conviction.expungement_result.time_eligibility.status:
+            arrest_dispositions = {'Dismissed', 'Acquitted', 'Dismissal', 'No Complaint'}
```

Comment:
A charge already knows if it is an arrest or a conviction via its public interface. By hard-coding these values here, we are duplicating code, and it has the potential to create hard to find bugs. For example: If the criteria for arrest changes in the future, we will have to change it in the charge class and remember to change it here.

Reply:
@NickSchimek ah ok, I'll make a change here.

Reply:
@aemrich I'm going to start working on issue #601 , which will subsume this check for different ruling strings, and express convicted/dismissed as a boolean or more standardized string.

Reply:
@wittejm ok, sounds good. I'll keep that in mind (I'm going to use convicted() method from the Charge class for now, but will change as you work on #601 )